src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -1,10 +1,18 @@
 {% trans_default_domain "locationview" %}
 
 {% if draftVersions %}
+    {{ form_start(draftActionsForm, {"action": path("ez_hybrid_platform_ui_content_view_draft_actions"), 'attr': {'class': 'ez-js-standard-form'}}) }}
```

Comment:
There looks to be some JS that listens to form submit and makes this an ajax request. To have it post in the normal way I've added this class: https://github.com/ezsystems/hybrid-platform-ui-core-components/blob/master/js/ez-platform-ui-app.js#L53